.github/workflows/contracts-ecdsa.yml

```diff
@@ -0,0 +1,203 @@
+name: Solidity ECDSA
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - main
+    paths:
+      - "solidity/ecdsa/**"
+  pull_request:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment for workflow execution"
+        required: false
+        default: "dev"
+      upstream_builds:
+        description: "Upstream builds"
+        required: false
+      upstream_ref:
+        description: "Git reference to checkout (e.g. branch name)"
+        required: false
+        default: "main"
+
+jobs:
+  contracts-detect-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      path-filter: ${{ steps.filter.outputs.path-filter }}
+    steps:
+      - uses: actions/checkout@v2
+        if: github.event_name == 'pull_request'
+
+      - uses: dorny/paths-filter@v2
+        if: github.event_name == 'pull_request'
+        id: filter
+        with:
+          filters: |
+            path-filter:
+              - './solidity/ecdsa/**'
+
+  contracts-lint:
+    needs: contracts-detect-changes
+    if: |
+      github.event_name == 'push'
+        || needs.contracts-detect-changes.outputs.path-filter == 'true'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./solidity/ecdsa
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14.x"
+          cache: "yarn"
+          cache-dependency-path: solidity/ecdsa/yarn.lock
+
+      - name: Install dependencies
+        run: yarn install --network-concurrency 1
```

Comment:
Have you tried just `yarn install`?
`--network-concurrency 1` was used as a workaround for failing installation in `random-beacon`, but maybe we wan't get the failure here in `ecdsa`. `--network-concurrency 1` may slow down installation a bit, so it may be worth to remove it if it's not needed.
Same applies to other places in the file where we use --network-concurrency 1`.

Reply:
🤦🏻 copy-paste 🤦🏻